docs/UserGuide.md

```diff
@@ -16,7 +16,55 @@ CS students are incredibly busy and hence having a command line interface (CLI)
 1. Ensure that you have Java 11 or above installed.
 1. Down the latest version of `DukeOfBooks` from [here](https://github.com/AY2223S2-CS2113-F10-4/tp).
 
-## Features 
+## Access Management
+The application requires user to login before performing other actions.
+
+### Login: `login`
+Login the system using username and password. 
+
+Format:
+* login -username [username] -password [password]
```

Comment:
Suggestion to follow lecture style annotation

login -username USERNAME -password PASSWORD